ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -48,7 +48,7 @@ const getFreqLinkPath = ({ chrom, pos, variant, value }) => {
   } else if (isRegion) {
     const posInt = parseInt(pos, 10)
     const endOffset = variant.end ? variant.end - variant.pos : 0
-    coords = `${chrom}-${posInt - 100}-${posInt + endOffset + 100}`
+    coords = `${chrom}-${Math.max(posInt - 100, 1)}-${posInt + endOffset + 100}`
```

Comment:
It was a bug that if `posInt` was smaller than 100 then the generated coordination was wrong.